#4073 updating kubeadm file. (#4083)
cri-o/cri-o: tutorials/kubeadm.md

Given you've set CIDR, and you've properly set the kubelet file, all you need to do is start crio (as defined [here](setup.md)), and run:
`kubeadm init --pod-network-cidr=$CIDR`

# Running kubeadm in an off line network

> We will assume that the user has installed CRI-O and alls necessary packages. We will also assume that all necessary components are configured and everything is working as expected. The user should have a private repo where the docker images are pushed. Sample of images fot version 1.18.2:

Reviewer: ```suggestion
We will assume that the user has installed CRI-O and all necessary packages. We will also assume that all necessary components are configured and everything is working as expected. The user should have a private repo where the container images are pushed. An example of container images for Kubernetes version 1.18.2:
```

```bash
$ kubeadm config images list --image-repository user.private.repo --kubernetes-version=v1.18.2
user.private.repo/kube-apiserver:v1.18.2
user.private.repo/kube-controller-manager:v1.18.2
user.private.repo/kube-scheduler:v1.18.2
user.private.repo/kube-proxy:v1.18.2
user.private.repo/pause:3.2
user.private.repo/etcd:3.4.3-0
user.private.repo/coredns:1.6.7
```

> The user needs to configure the [registries.conf](https://www.mankier.com/5/containers-registries.conf) file.

Reviewer: I'd prefer pointing to the source, even though it's less pretty.
```suggestion
The user needs to configure the [registries.conf](https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md) file.
```

Author: @TomSweeneyRedHat I agree with your comments. I did not noticed those small minor mistakes as well. Thank you for pointing them out :).

Let me know if there is anything else that I can help with.

BR / Thanos

Next the user should reload and restart the CRI-O service to load the configurations.

> The user should not forget to add the cgroup driver. The following are samples of parameters for the configuration file as documented also at [Configure cgroup driver used by kubelet on control-plane node](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node):

Reviewer: This is not at all like the first review.  Did you mean to move to this?  I preferred the prior review fwiw.

Author: I will update it I though that the comments from @haircommander was to remove the whole line. I will revert.

Author: Done
